chainer/functions/noise/dropout.py

```diff
@@ -12,6 +12,7 @@ class Dropout(function.Function):
     """Dropout regularization."""
 
     def __init__(self, dropout_ratio):
+        assert 0.0 <= dropout_ratio and dropout_ratio <= 1.0
```

Comment:
- It's better to raise `ValueError`, because this is to check for external specification, not internal implementation.
- You can write `0.0 <= dropout_ratio <= 1.0`.